### PR TITLE
Task 2,3,4

### PR DESCRIPTION
**Pull request by MaxKostanyan**

### CODE REVIEW

File: Task_2/src/Task_2.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        System.out.println("\n" + boarder + "\nTask 3\n" + "Create an array and fill it with odd numbers from -20:20\n");
+
+        int[] minusTwentyToTwenty = new int[20];
+        for (int i = 0; i < minusTwentyToTwenty.length; i++) {
+            minusTwentyToTwenty[i] = i + 1;
+            if (minusTwentyToTwenty[i] % 2 != 0) {
+                System.out.println("Positive odd numbers " + minusTwentyToTwenty[i]);
+                System.out.println("Negative odd numbers " + - + minusTwentyToTwenty[i]); // should be discussed :)))
+            }
+        }
```
Comment (Karengrigoryan060) [inline Task_2/src/Task_2.java:30]: I am not sure this is working correctly.

File: Task_2/src/Task_2.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+        System.out.println("\n" + boarder + "\nTask 4\n" + "Create an array and fill it. Print all elements which can be divided by 5.\n");
+
+        int[] divideToFive = new int[100];
+        for (int i = 0; i < divideToFive.length; i++) {
+            divideToFive[i] = i + 1;
+            if(divideToFive[i] % 5 == 0) {
+                System.out.println(divideToFive[i]);
+            }
+        }
```
Comment (Karengrigoryan060) [inline Task_2/src/Task_2.java:40]: this implementation is not what was expected by the task. 
For first you should fill the list
After print the required data from the list

File: Task_2/src/Task_2.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        System.out.println("\n" + boarder + "\nTask 5\n" + "Create an array and fill it. Print all elements which are between 24.12 and 467.23.\n");
+
+
+        int[] betweenTwentyFourAndFourHundredSeven = new int[600];
+        for (int i = 23; i < betweenTwentyFourAndFourHundredSeven.length; i++) {
+            betweenTwentyFourAndFourHundredSeven[i] = i + 1;
+            if (betweenTwentyFourAndFourHundredSeven[i] > 24.12 && betweenTwentyFourAndFourHundredSeven[i] < 467.23) {
+                System.out.println(betweenTwentyFourAndFourHundredSeven[i]);
+            }
+        }
```
Comment (Karengrigoryan060) [inline Task_2/src/Task_2.java:51]: the same as above

**Review by Karengrigoryan060 — COMMENTED** [review]
Please, try to do tasks step by step,

File: Transactions/src/BubbleSort.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
+public class BubbleSort {
+   static void bubbleSort ( int[] arr) {
+        int n = arr.length;
+        int temp = 0;
```
Comment (armanayvazyan) [inline Transactions/src/BubbleSort.java:4]: Good job, but what is the meaning of defining temp outside the loop?

Comment (MaxKostanyan) [reply Transactions/src/BubbleSort.java:4]: I guess there is no difference, will check

**Pull request merged**